src/main/utils/vencordLoader.ts

```diff
@@ -13,7 +13,12 @@ import { downloadFile, simpleGet } from "./http";
 
 const API_BASE = "https://api.github.com";
 
-const FILES_TO_DOWNLOAD = ["vencordDesktopMain.js", "preload.js", "vencordDesktopRenderer.js", "renderer.css"];
+const FILES_TO_DOWNLOAD = [
+    "vencordDesktopMain.js",
+    "preload.js",
```

Comment:
a

Reply:
actually, maybe deduplicate this code by move this to a shared file and import, i think src/main/constants